_data/sidebars/main.yml

```diff
@@ -14,19 +14,35 @@ subitems:
         url: /life_cycle
       - title: Three-tier model
         url: /three_tier_view
-      - title: Quality & FAIR
-        url: /quality_fair
-  - title: Research software quality
-    description: Indicators & principles for research software quality
+      - title: Quality
+        url: /quality
+  - title: Quality dimensions & indicators
+    description: Dimensions and indicators of research software quality
     url: /rs_quality
     image_url: /assets/img/font-awesome-free/star-solid.svg
     subitems:
       - title: Compatibility
-        url: /dimension-compatibility
+        url: /rs_quality#compatibility
```

Comment:
Is the idea to have everything on one page in the end? If so, we can remove the rs_quality folder, right? And maybe to have a slightly cleaner navigation menu, we can move the Quality dimensions & indicators link into the Research Software submenu. Perhaps the separate links to sections in this page make more sense when the texts become longer.

Reply:
Yes, I have just removed individual dimension pages and genereating eveyrthing from the data file. We can get rid of that whole fodler.